examples/dfk2wan.jl

```diff
@@ -51,29 +48,80 @@ model = model_PBE(lattice,atoms)
 
 kgrid = [4,4,4]
 Ecut = 20
-basis = PlaneWaveBasis(model, Ecut; kgrid=kgrid, use_symmetry = false)
+basis = PlaneWaveBasis(model, Ecut; kgrid=kgrid, optimize_fft_size = true, use_symmetry=false)
 
-scfres = self_consistent_field(basis, tol=1e-12, n_bands = 4, n_ep_extra = 0 );
+scfres = self_consistent_field(basis, tol=1e-12, n_bands = 12, n_ep_extra = 0);
 
-# We may now create the `.win` file. In addition of information on the system
-# (via basis, scfres and kgrid) the functon asks for a name (here "Si") and
-# the number of wanted MLWFs. For isolated bands, this number is the number
-# of bands, i.e. four in our case.
+# We may now create the `.win` file. In addition of informations on the system
+# (via `basis`, `scfres` and `kgrid`) the functon asks for a name (here "Si") and
+# the number of wanted MLWFs; 8 in our case. The keyword `bands_plot = true`
+# adds the ``k`` point path to the win file, needed to plot the bands structure.
+# We also add as keyword arguments the parameters of the disentanglement
+# procedure. Any other argument can be added as such.
 
-dftk2wan_win_file("Si",basis,scfres,4, kgrid = kgrid)
+dftk2wan_win_file("Si", basis, scfres, kgrid, 8;
+                  bands_plot=true, num_print_cycles=50, num_iter=500,
+                  dis_win_max       = "17.185257d0",
+                  dis_froz_max      =  "6.8318033d0",
+                  dis_num_iter      =  120,
+                  dis_mix_ratio     = "1d0")
 
-# Then, we perfom a preproprocessing task via Wannier90 to generate
+# We then perfom a preproprocessing task via Wannier90 to generate
 # the `Si.nnkp` file. This is simply done by :
 
-PathToWannier90/wannier90.x -pp Si
+#md #```@raw bash
+#md #$ PathToWannier90/wannier90.x -pp Si
```

Comment:
There is now https://github.com/JuliaBinaryWrappers/wannier90_jll.jl, so you can (soon) just install and use Wannier90 for the Julia repl and from Julia scripts. E.g. for this you would:
```
using wannier90_jll
wannier90() do exe
    run(`$exe -pp Si`)
end
```

Reply:
Oh wow that's cool. Did you do this?

Reply:
Yes.

Reply:
Haha, week-ends I do performance optimizations, you do devops, something is wrong with us ;-) So basically you did this https://github.com/JuliaPackaging/Yggdrasil/pull/2153/files and then it works all by itself? Neat!